scripts/mbedtls_dev/generate_server9_bad_saltlen.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
```

Comment:
Since this is an executable script, not a library module, it should be in `test/scripts`.